Update sitemap provider (#912)
sulu-io/sulu-docs: cookbook/sitemap-provider.rst

            return $result;
        }

        public function getAlias(): string
        {
            return 'myalias';
        }

        public function createSitemap($scheme, $host): Sitemap
        {
            return new Sitemap($this->getAlias(), $this->getMaxPage($scheme, $host));
        }

        public function getMaxPage($scheme, $host): int
        {
            if ($host !== 'example.org') {
                // If the pages are only for a specific host.
                return 0;
            }

            return (int) ceil($this->repository->countForSitemap() / self::PAGE_SIZE);
        }
    }

> If you have disabled autoconfiguration for this service, add the tag manually:

Reviewer: ```suggestion
If you have disabled autowiring, add the tag manually:
```
